consensus/tool/src/main.rs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! A cli tool for interrogating a mobilecoin consensus node.
+
+#![deny(missing_docs)]
+
+use clap::{Parser, Subcommand};
+use grpcio::{ChannelBuilder, EnvBuilder};
+use mc_common::logger::{create_app_logger, o};
+use mc_consensus_api::{consensus_common_grpc::BlockchainApiClient, empty::Empty};
+use mc_util_grpc::ConnectionUriGrpcioChannel;
+use mc_util_uri::ConsensusClientUri;
+use std::{sync::Arc, time::Duration};
+
+/// Command line config, set with defaults that will work with
+/// a standard mobilecoind instance
+#[derive(Clone, Debug, Parser)]
+#[clap(
+    name = "mc-consensus-tool",
+    about = "A cli tool for interrogating one or more consensus nodes"
+)]
+pub struct Config {
+    /// Command to use
+    #[clap(subcommand)]
+    pub tool_command: ToolCommand,
+
+    /// Consensus_uri's to connect to
+    #[clap(global = true)]
+    pub consensus_uris: Vec<ConsensusClientUri>,
+}
+
+/// Commands that the tool recognizes
+#[derive(Clone, Debug, Subcommand)]
+pub enum ToolCommand {
+    /// Status: Prints the network status, including block count and block
+    /// version
+    Status,
+    /// Wait-for-quiet: Blocks until the network is 'quiet'. This means that the
+    /// block count doesn't move for some time.
+    WaitForQuiet {
+        /// Number of seconds the network must stop moving for. Defaults to 20.
+        #[clap(long)]
```

Comment:
Any reason not to use `default_value = "20"` here?

Reply:
I'll try again, I was using `default_value_t` because that was documented and it didn't seem to work with `derive(Subcommand)`. But maybe this version will work here.